package.json

```diff
@@ -7,15 +7,15 @@
   },
   "author": {
     "name": "Cogworks",
-    "url": "http://www.thecogworks.com/"
+    "url": "http://www.wearecogworks.com/"
   },
-  "readme": "A simple and intuitive package which allows editors to find and replace content in Umbraco 7.",
+  "readme": "A simple and intuitive package which allows editors to find and replace content in Umbraco 8.x",
```

Comment:
I tried to avoid setting up the version in e.g. Sitelock release and just left there a note that it's for Umbraco. There is a version for v7, don't know how we should distribute it and if we should think about this readme only for the version of the package or the whole package in general? 🤔

Reply:
Any proposals ?